packages/opentelemetry-exporter-collector-grpc/test/CollectorMetricExporter.test.ts

```diff
@@ -20,7 +20,11 @@ import {
   ValueObserver,
   ValueRecorder,
 } from '@opentelemetry/api-metrics';
-import { ConsoleLogger, LogLevel } from '@opentelemetry/core';
+import {
+  DiagConsoleLogger,
+  DiagLogLevel,
+  createLogLevelDiagLogger,
```

Comment:
So many diag logger exports, I had thought these would be for internal use. It is going to be confusing to users to have 7 exports all to do with logging.

Reply:
Generally, I would not expect end-users to need to import ```createLogLevelDiagLogger``` and reviewing this specific case it doesn't. This was just the replacement for the previous test usage of ```new ConsoleLogger(LogLevel.DEBUG);``` which it's really not using.

But more generally, they would almost never require 7 and in most cases they will only need to use ```getDiagLoggerFromConfig()``` unless they want to use somethin specific.

Reply:
I wasn't meaning a user would need to import that many things. I mean the existence of so many things will make it difficult for the user to figure out which 1 or 2 they should be using.

Reply:
After looking through everything exported and the docstrings, i think this is less of a problem than I had initially thought. going to mark this as resolved.